ingestion-beam/src/main/java/com/mozilla/telemetry/contextualservices/ParseReportingUrl.java

```diff
@@ -146,15 +190,19 @@ public Result<PCollection<PubsubMessage>, PubsubMessage> expand(
                 attributes.get(Attribute.GEO_SUBDIVISION1));
             urlParser.addQueryParam(ParsedReportingUrl.PARAM_OS_FAMILY,
                 getOsParam(attributes.get(Attribute.USER_AGENT_OS)));
-            urlParser.addQueryParam(ParsedReportingUrl.PARAM_FORM_FACTOR, "desktop");
+            urlParser.addQueryParam(ParsedReportingUrl.PARAM_FORM_FACTOR,
+                interaction.getFormFactor());
 
             urlParser.addQueryParam(ParsedReportingUrl.PARAM_DMA_CODE,
                 message.getAttribute(Attribute.GEO_DMA_CODE));
           }
 
           // We only add these dimensions for topsites clicks, not quicksuggest per
+          // We are also limiting this to desktop.
           // https://bugzilla.mozilla.org/show_bug.cgi?id=1738974
```

Comment:
additional comment should probably be added at the end, instead of in the middle:
```suggestion
          // We only add these dimensions for topsites clicks, not quicksuggest per
          // https://bugzilla.mozilla.org/show_bug.cgi?id=1738974
          // We are also limiting this to desktop.
```